inlong-agent/conf/log4j2.xml

```diff
@@ -20,7 +20,7 @@
 <configuration status="WARN" monitorInterval="30">
     <Properties>
         <property name="basePath">${sys:agent.home}/logs</property>
-        <property name="log_pattern">%d{yyyy-MM-dd HH:mm:ss.SSS} -%5p ${PID:-} [%15.15t] %-30.30C{1.} : %m%n</property>
+        <property name="log_pattern">%d{yyyy-MM-dd HH:mm:ss.SSS} -%5p ${PID:-} [%15.15t] %-30.30C{1.}:%L %m%n</property>
```

Comment:
Are there any other log configurations of the agent module that need to be modified?

Reply:
The agent module is not adjusted first, and the logs that other modules do not print line numbers are changed
<img width="667" alt="wecom-temp-b576ae7825a12152918685d5fa3d7d8b" src="https://user-images.githubusercontent.com/20356765/176813628-231daad6-f043-4f39-b4eb-4b18e96521d7.png">